clusterloader2/framework/util/util.go

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
I'm not huge fan of "util.go" files, because they hugely become a mix of everything.

Those are functions that implement client operations on objects. So maybe:
client/objects.go
?

Or sth like that?